web_src/js/components/DiffCommitSelector.vue

```diff
@@ -18,17 +24,17 @@
           </div>
         </div>
         <!-- only show the show changes since last review if there is a review AND we are commits ahead of the last review -->
-        <div v-if="lastReviewCommitSha != null && commitsSinceLastReview > 0" class="vertical item gt-df gt-fc gt-gap-2 gt-border-secondary-top" @click="changesSinceLastReviewClick()">
+        <div v-if="lastReviewCommitSha != null" role="menuitem" tabindex="-1" class="vertical item gt-df gt-fc gt-gap-2 gt-border-secondary-top" :class="{disabled: commitsSinceLastReview === 0}" @keydown.enter="changesSinceLastReviewClick()" @click="changesSinceLastReviewClick()">
```

Comment:
Instead of simulating buttons, we might as well use `<button>`s directly (if we replace the inner `<div>`s with `<span>`s).